docs/topics/managedcluster.md

```diff
@@ -0,0 +1,147 @@
+# Managed Clusters (AKS)
+- **Feature status:** Experimental
+- **Feature gate:** AKS=true,MachinePool=true
+
+Cluster API Provider Azure (CAPZ) experimentally supports managing Azure
+Kubernetes Service (AKS) clusters. CAPZ implements this with three
+custom resources:
+- AzureManagedControlPlane
+- AzureManagedCluster
+- AzureManagedMachinePool
+
+The combination of AzureManagedControlPlane/AzureManagedCluster
+corresponds to provisioning an AKS cluster. AzureManagedMachinePool
+corresponds one-to-one with AKS node pools. This also means that
+creating an AzureManagedControlPlane requires defining the default
+machine pool, since AKS requires at least one system pool at creation
+time.
+
+## Deploy with clusterctl
+
+A clusterctl flavor exists to deploy an AKS cluster with CAPZ. This
+flavor requires the following environment variables to be set before
+executing clusterctl.
+
+```bash
+# Kubernetes values
+export CLUSTER_NAME="foobar"
+export WORKER_MACHINE_COUNT=2
+export KUBERNETES_VERSION="1.17.4" # NOTE: no preceding "v", unlike normal for CAPI.
+
+# Azure values
+export AZURE_LOCATION="southcentralus"
+export AZURE_RESOURCE_GROUP="${CLUSTER_NAME}"
+export AZURE_SUBSCRIPTION_ID="$(cat ~/sp.json | jq -r .subscriptionId | tr -d '\n')"
+export AZURE_NODE_MACHINE_TYPE="Standard_D8s_v3"
+```
+
+Execute clusterctl to template the resources, then apply to a management cluster:
+
+```bash
+# replace v0.4.4 with the appropriate version you're using
```

Comment:
Do we need to specify the capz version explicitly? or could this just be `--infrastructure azure` to use latest?

Reply:
--infrastructure azure didn't work properly for me, i'll try this again maybe?

Reply:
We can keep it as is for now until we get --feature-flags. :)